test/avrora/encoder_test.exs

```diff
@@ -732,4 +788,8 @@ defmodule Avrora.EncoderTest do
   defp payment_json_schema do
     ~s({"namespace":"io.confluent","name":"Payment","type":"record","fields":[{"name":"id","type":"string"},{"name":"amount","type":"double"}]})
   end
+
+  defp numeric_transfer_json_schema do
+    ~s({"namespace":"io.confluent","name":"numeric_transfer","type":"record","fields":[{"name":"link_is_enabled","type":"boolean"},{"name":"updated_at","type":"int"},{"name":"updated_by_id","type":"int"}]})
```

Comment:
May I ask you to follow the guideline for naming (it's not so important here, but oh my OCD) and rename the schema name to `NumericTransfer`

Reply:
done